### ISSUE TITLE
`cancelDisplayedNotifications` also cancels scheduled notifications on Android

### ISSUE THREAD

**Issue by mars-lan** [notifee/react-native-notifee#349]
Based on the name &  [documentation](https://notifee.app/react-native/reference/canceldisplayednotifications), I assume the function only cancles displayed/delivered notifications. But it seems that on Android it also cancels all scheduled (but not yet triggered) notifications. Here is an example that demostrates the bug. By commenting out the `cancelDisplayedNotifications` line the notification will fire as expected after 3s.

```
import React from 'react';
import { AppRegistry } from 'react-native';
import notifee, { TriggerType } from '@notifee/react-native';


async function init() {
  await notifee.createChannel({
    id: 'channel',
    name: 'channel',
  });

  await notifee.createTriggerNotification({
    title: 'Title',
    body: 'Body',
    android: {
      channelId: 'channel',
    },
  }, {
    type: TriggerType.TIMESTAMP,
    timestamp: (new Date()).getTime() + 3000,
  });

  // Comment out the following line to make the above trigger notification work
  await notifee.cancelDisplayedNotifications();
}

init();

function Test() {
  return null;
}

AppRegistry.registerComponent('TrayMinder', () => Test);
```

**Comment by helenaford** [comment]
@mars-lan oh, is that since the latest update? I'll take a look into this ASAP, thanks for bringing it to our attention.

**Comment by mars-lan** [comment]
@helenaford Tested on both 1.9.2 & 1.10.0.

**Comment by helenaford** [comment]
@mars-lan thanks for confirming. I can see the issue and have a release shortly.

**Comment by helenaford** [comment]
@mars-lan should be fixed now (release 1.10.1) 🙏 Release notes will be live soon (https://notifee.app/react-native/docs/release-notes)

**Comment by mars-lan** [comment]
Thanks for the quick fix!
